pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -6,6 +6,8 @@
     <_PreReleasePackageVersion>$(PackageVersion)</_PreReleasePackageVersion>
     <PackageVersion>2.1.0</PackageVersion>
     <ServiceModelVersion>4.4.1</ServiceModelVersion>
+    <!-- We don't need to harvest the stable packages to build this -->
+    <HarvestStablePackage>false</HarvestStablePackage>
```

Comment:
I kind of feel that this should be set automatically if there are no HarvestIncludePath defined

Reply:
I've thought about that a couple times but we also harvest the set of supported frameworks which is still good for most packages to do validation even if they aren't harvesting assets.

Reply:
Should the stable versiosn for this guys be updated to 2.0.1 in the packageIndex.json now that you're updating the index for other packages?

Reply:
No one is really referencing the version so it doesn't need to be updated in the index. It doesn't hurt but I simply updated to the set of packages that were stable in the last published set in corefx https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.1/Latest_Packages.txt. For some reason the compat packages weren't listed as stable there yet.